regular_expressions_lookaheads/regular_expressions_lookaheads.md

```diff
@@ -0,0 +1,254 @@
+<!--
+
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 1.0.0
+language: en
+narrator: US English Female
+mode: Textbook
+previous_sequential_module: regular_expressions_basics
+
+title: Regular Expressions, Intermediate Level
+
+comment:  Use regular expressions, or regex, for complex pattern matching involving lookaheads.
+
+long_description: Learn to compose intermediate regular expressions using lookahead syntax, in order to identify text that matches patterns you describe.
+
+estimated_time_in_minutes: 30
+
+@pre_reqs
+Learners should have some experience composing and using simple regular expressions (regex).  For an introduction to regular expression concepts, consider the [Demystifying Regular Expressions](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_regular_expressions/demystifying_regular_expressions.md#1) module.  To learn how to compose and use simple regular expressions, consider the [Regular Expressions Basics](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/regular_expressions_basics/regular_expressions_basics.md#1) module.
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Explain the difference between "moving" ahead and "looking" ahead in regular expression parsing
+- Explain why a "lookahead" can be useful in a regular expression
+
+@end
+good_first_module: false
+coding_required: true
+coding_level: intermediate
+sequence_name: regex
+previous_sequential_module: regular_expressions_anchors_boundaries
+@sets_you_up_for
+
+@end 
+@depends_on_knowledge_available_in
+- demystifying_regular_expressions
+- regular_expressions_basics
+- regular_expressions_groups
+- regular_expressions_anchors_boundaries
```

Comment:
```suggestion
```
I think it doesn't actually depend on anything in the anchors/boundaries module?